app/src/main/res/layout/item_planning.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
```

Comment:
You either use `LinearLayout` OR `ConstraintLayout` - in this case  one of them is unnecessary and it's easier to remove the `ConstraintLayout` but I want you to remove the `LinearLayout` and try to make this layout in `ConstraintLayout` (it's more complex but used widely nowadays so you have to know how to deal with it)